.neutrinorc.js

```diff
@@ -21,7 +21,7 @@ module.exports = {
       popup: {
         entry: 'src/popup',
         webext: {
-          type: 'browser_action'
+          type: 'action'
```

Comment:
Why this is renamed?

Reply:
Google merged browser_action and page_action in v3, see https://developer.chrome.com/docs/extensions/mv3/intro/mv3-migration/#action-api-unification

Reply:
I think we should keep both for v2 support.

Reply:
that's the case, it supports both :)